doc/user_guide/supported_formats/mrc.rst

```diff
@@ -56,13 +56,20 @@ MRC Format (Direct Electron)
 Loading from Direct Electron's ``.mrc`` as well as reading the metadata from the .txt file
 saved by the software is supported by passing the ``metadata_file`` argument to the
 ``file_reader`` function. The ``metadata_file`` argument can be a string or a file-like
-object.
+object. Additionally, the ``metadata_file`` argument can be automatically inferred.  This requires
+that the file name is of the form ``uniqueid_suffix_movie.mrc`` and that the metadata file is
+named ``uniqueid_suffix_info.txt``.
 
 This will automatically set the navigation shape based on the ``Scan - Size X`` and                                                   = 256
-``Scan - Size Y`` as well as the ``Scan - Repeats`` and ``Scan - Point Repeats``
-parameters in the metadata file. The navigation shape can be overridden
+``Scan - Size Y``parameters in the metadata file. The navigation shape can be overridden
```

Comment:
Use full word instead of "=256" because it doesn't read well.

There is a formatting issue:
![image](https://github.com/user-attachments/assets/2f1f5219-44ce-423b-8527-5caf2407b4ac)

Add examples to illustrate how to load the metadata file (automatically or by specifying)?

Reply:
Examples should be added.